### ISSUE TITLE
✨ Delete a connector

### ISSUE THREAD

**Issue by tjarbo** [tjarbo/discord-moodle-bot#146]
> **As a** user
> **I want** delete a connector
> **because** I do not use the platform anymore

> ⚠️ Blocked by #144 

## Description:
At some point, the user wants to delete a connector, and the user can do this, by clicking the Trash Icon in the bottom right corner of the connector panel. After the user has confirmed, the connector will be deleted. It is possible to have no connector registered!

### 🟢 In scope:
* Input validation!
* Implement `DELETE /api/connectors/:id ` to delete a give connector
* Give warning at frontend as second confirm, if the user wants to delete the last connector 

### 🔴 Not in scope:
* Update Documentation

**Comment by tjarbo** [comment]
Done in #152
